dashboard/app/views/levels/editors/_gamelab.html.haml

```diff
@@ -97,11 +92,12 @@
   = f.text_area :validation_code
 
 .field
-  =f.label 'Animations are paused by default'
+  .aligned-input-group
+    = boolean_check_box f, :pause_animations_by_default
+    =f.label 'Animations are paused by default'
```

Comment:
is there a reason these nor the `_gamelab_jr.html.haml` changes aren't using the partial?